addons/web/static/src/core/hotkeys/hotkey_service.js

```diff
@@ -34,10 +34,19 @@ export const hotkeyService = {
         let nextToken = 0;
         let overlaysVisible = false;
 
-        browser.addEventListener("keydown", onKeydown);
-        browser.addEventListener("keyup", removeHotkeyOverlays);
-        browser.addEventListener("blur", removeHotkeyOverlays);
-        browser.addEventListener("click", removeHotkeyOverlays);
+        bindDocument(browser);
+
+        /**
+         * Bind events to a document.
+         *
+         * @param {Document} document
+         */
+        function bindDocument(document) {
```

Comment:
unused argument, how this works ?

Reply:
Corrected, I don't know what happened with the commit.